catroid/src/main/java/org/catrobat/catroid/utils/FlashUtil.java

```diff
@@ -123,6 +136,11 @@ public static void flashOn() {
 
 	public static void flashOff() {
 		Log.d(TAG, "flashOff()");
```

Comment:
another log that can be removed (I know it's not part of your commit, but please do it anyway).